features/configuration_files/masking_smells.feature

```diff
@@ -34,7 +34,17 @@ Feature: Masking smells using config files
       """
 
   Scenario: provide extra masking inline in comments
-    Given the smelly file 'smelly_with_inline_mask.rb'
+    Given a file named "smelly_with_inline_mask.rb" with:
```

Comment:
Why do you inline this but not others?

Reply:
The reason for that was that the configuration file for this scenario was *only* used for this scenario while the other scenarios in this feature were talking about smelly.rb:

```Cucumber
Given the smelly file 'smelly.rb'
```

which is *always* talking about the same file. I guess you could call this the reference file for smelly code. 
I could inline all of those calls but that would be a *lot* of duplication and if we ever want to change our understanding of this reference file a huge. I know it's a tradeoff, but for now I'd not inline "smelly.rb" but only the files which we use for one specific scenario only. Wdyt?

Reply:
Ok, that makes perfect sense.